modules/microshift-observe-debug-etcd-server.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+//* microshift_support/microshift-etcd.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-observe-debug-etcd-server_{context}"]
+= Observe and debug the {product-title} etcd server
+
+You can gather `journalctl` logs to observe and debug the etcd server logs.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+
+* You have run the command `systemctl start microshift` as part of the {product-title} start-up.
+
+.Procedure
+
+* To get the logs for etcd, run the following command:
++
+[source,terminal]
+----
+$ journalctl -u microshift-etcd
```

Comment:
```suggestion
$ journalctl -u microshift-etcd.scope
```
Also, it needs to run as privileged user ( `$` suggests regular user, is that the case?)

Reply:
Yes, the command must be `$ sudo journalctl -u microshift-etcd.scope`